lib/structuraid_core/designers/footings/steps/centric_isolated/check_reinforcement.rb

```diff
@@ -0,0 +1,75 @@
+require 'interactor'
+
+module StructuraidCore
+  module Designers
+    module Footings
+      module Steps
+        module CentricIsolated
+          # Checks supplied reinforcement: ratio and spacing
+          class CheckReinforcement
+            include Interactor
+
+            # @param footing [StructuraidCore::Elements::Footing] The footing to be designed
+            # @param analysis_results [Hash] The analysis results
+            # @param design_code [StructuraidCore::DesignCodes] The design code to be used
+            # @param analysis_direction [Symbol] The direction for which the analysis has to be run. Should be either :length_1 or :length2
+            # @param support_type [Symbol or String] The support type: :over_soil or :over_piles
+            # @param steel [StructuraidCore::Materials::Steel] The rebar's material
+            def call
+              check_reinforcement_ratio
+              check_reinforcement_spacing
+            end
```

Comment:
checking reinforcement consists of two requirements:
- the reinforcement must provide ate least the required ratio
- the reinforcement spacing must be less or equal to maximum spacing